openwisp_users/tests/test_admin.py

```diff
@@ -201,6 +201,149 @@ def test_operator_changelist_superuser_column_hidden(self):
+        self.assertContains(response, 'selected>operator-org1</option>')
+        self.assertContains(response, 'selected>operator-org2</option>')
+
+    def test_operator_can_see_organization_add_user(self):
+        org1 = self._create_org(name='operator-org1')
+        org2 = self._create_org(name='operator-org2')
+        operator = self._create_operator()
+        org_permissions = Permission.objects.filter(codename__endswith='organization_user')
+        operator.user_permissions.add(*org_permissions)
+        options1 = {
+            'organization': org1,
+            'is_admin': True,
+            'user': operator
+            }
+        options2 = {
+            'organization': org2,
+            'is_admin': False,
+            'user': operator
+            }
+        self._create_org_user(**options1)
+        self._create_org_user(**options2)
+        self.client.force_login(operator)
+        response = self.client.get(reverse('admin:openwisp_users_user_add'))
+        self.assertContains(response, 'operator-org1</option>')
+        self.assertNotContains(response, 'operator-org2</option>')
+
+    def test_operator_change_organization(self):
+        org1 = self._create_org(name='test-org1')
+        org2 = self._create_org(name='test-org2')
+        default_org = Organization.objects.get(name='default')
+        operator = self._create_operator()
+        org_permissions = Permission.objects.filter(codename__endswith='change_organization')
+        operator.user_permissions.add(*org_permissions)
+        options1 = {
+            'organization': org1,
+            'is_admin': True,
+            'user': operator
+        }
+        options2 = {
+            'organization': org2,
+            'is_admin': False,
+            'user': operator
+        }
+        self._create_org_user(**options1)
+        self._create_org_user(**options2)
+        self.client.force_login(operator)
+        response = self.client.get(reverse('admin:openwisp_users_organization_change',
+                                           args=[org1.pk]))
+        self.assertContains(response,
+                            '<input type="text" name="name" value="{0}"'.format(org1.name))
+        response = self.client.get(reverse('admin:openwisp_users_organization_change',
+                                           args=[default_org.pk]))
+        self.assertContains(response,
+                            '<input type="text" name="name" value="{0}"'.format(default_org.name))
+        response = self.client.get(reverse('admin:openwisp_users_organization_change',
+                                           args=[org2.pk]))
+        # With django versions >= 2.1, response have a status code of 200
+        # Where as in versions < 2.1, response have a status code of 403
+        # Thus, we have to ensure the required test is carried out in each case
+        from django import __version__ as version
+        if version >= '2.1':
```

Comment:
do it as it's suggested here: https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python